Lob-API-public.yml

```diff
@@ -111,6 +111,24 @@ tags:
   - name: Intl Verifications
     description: Address verification for non-US addresses
 
+  - name: Intl Verifications Test Environment
+    x-traitTag: true
+    description: |
+      When verifying international addresses, you'll likely want to test against
+      a wide array of addresses to ensure you're handling responses correctly.
+      With your test API key, requests that use specific values for `primary_line`
+      let you explore the responses to many types of addresses:
+
+      | DELIVERABILITY OF SAMPLE RESPONSE | SET `primary_line` TO    |
+      |-----------------------------------|--------------------------|
+      | `deliverable`                     | deliverable              |
+      | `deliverable_missing_info`        | deliverable missing info |
+      | `undeliverable`                   | undeliverable            |
+      | `no_match`                        | no match                 |
+
+      See the `test` request & response examples under [Intl Verification Examples](#operation/intl_verification) within the
+      Verify an international address section in Intl Verifications.
+
```

Comment:
Just wanted to ensure you actually intended to change the descrip here and didn't just forget about the original paragraphs. 
![image](https://user-images.githubusercontent.com/45194378/125851501-4caf70c7-8c9d-4eb7-bfee-75da98e933da.png)

Reply:
Good catch! I forgot to add in that part 🤦🏽